kornia/feature/dedode/dedode.py

```diff
@@ -41,7 +42,7 @@ class DeDoDe(Module):
         amp_dtype: The automatic mixed precision desired.
 
     Example:
-        >>> dedode = DeDoDe.from_pretrained(detector_weights="L-upright", descriptor_weights="B-upright")
+        >>> dedode = DeDoDe.from_pretrained(detector_weights="L-C4-v2", descriptor_weights="B-upright")
```

Comment:
Could you add it here for testing too? https://github.com/kornia/kornia/blob/b0995bdce3b04a11d39e86853bb1de9a2a438ca2/tests/feature/test_dedode.py#L45